dictionary/contexts.csv

```diff
@@ -21,3 +21,4 @@ NEWHORIZONS,New Horizons Mission,OPUS_GENERAL
 NHLORRI,New Horizons Long-Range Reconnaissance Imager (LORRI),NEWHORIZONS
 NHMVIC,New Horizons Multispectral Visible Imaging Camera (MVIC),NEWHORIZONS
 OPUS_PRODUCT_TYPE,OPUS Product Types,NULL
+MULT_MISSION,OPUS Mission Ids,NULL
```

Comment:
This isn't needed for now since we never actually use the contexts table. Eventually we'll want to add contexts automatically as the mult tooltips are loaded.

Reply:
There is a constraint that requires the context field in definitions to have a foreign key to contexts table name field. If we remove the entry in context.csv, we will get the following error when running import.
```
Failed to insert row into "imp_definitions": Cannot add or update a child row: a foreign key constraint fails (`opus3_test_group_inst_id`.`imp_definitions`, CONSTRAINT `imp_definitions_ibfk_1` FOREIGN KEY (`context`) REFERENCES `imp_contexts` (`name`) ON UPDATE CASCADE)
```

Reply:
OK, but then this isn't the way to do it because you'd have to add a new context to this file every time you wanted to add tooltips to a new mult. That is an unnecessary dependency. Instead, import_dictionary should create the contexts for each mult context as it's needed.